core/src/main/scala/kafka/server/BrokerToControllerRequestManager.scala

```diff
@@ -125,14 +125,18 @@ abstract class AbstractBrokerToControllerRequestManager[Item <: BrokerToControll
   private def sendRequest(itemsToSend: Seq[Item]): Unit = {
     val brokerEpoch = brokerEpochSupplier.apply()
     val request = buildRequest(itemsToSend, brokerEpoch)
-    debug(s"Sending to controller $request")
+    info(s"Sending to controller ${request.getClass.getName} of ~${request.toString.length} bytes (${itemsToSend.length} items)")
```

Comment:
What is estimated peak brokerToController request rate? If the peak rate is high, this log maybe to much.

Reply:
It's hard to tell without doing a full rebalance (which was already done before these changes, and we don't have an "unbalance" command to run), but the rate of these logs relative to all others when doing a single-broker bounce was tiny, to the point it was hard to spot them in scrollback even with color-coding. Plus, they're tiny. So I don't think it's a concern at all.